array.js

```diff
@@ -187,35 +187,36 @@ console.log(doMath(22, 2, '^'));
 /* 3. Написати функцію заповнення даними користувача двомірного масиву. Довжину основного масиву і внутрішніх масивів задає користувач. Значення всіх елементів всіх масивів задає користувач. */
 
 const addInfoToArray = () => {
-	const widthMainArray = prompt('Input width main array'); // 2
-	const widthSecondaryArray = prompt('Input width secondary array'); //2
+	const widthMainArray = +prompt('Enter the length the main array');
+	const widthSecondaryArray = +prompt('Enter the length the internal array in the main array');
```

Comment:
Enter the length `of` the main array

Enter the length `of` the internal array in the main array